pbspro.spec

```diff
@@ -50,9 +50,11 @@ Name: %{pbs_name}
 Version: %{pbs_version}
 Release: %{pbs_release}
 Source0: %{pbs_dist}
+%if 0%{?opensuse_bs}
```

Comment:
The problem was identified while building and testing this ticket, but it's not strictly related. Seemed like a lot of overhead to file a separate ticket, so I just made the change here.